aiida_lsmo/__init__.py

```diff
@@ -4,6 +4,4 @@
 AiiDA workflows for the LSMO laboratory at EPFL
 """
 
-from __future__ import absolute_import
-
 __version__ = "1.0.0a2"
```

Comment:
@danieleongari You'll probably want to update that

Reply:
Sorry I forgot to push: @yakutovicha can not share soon his checklist so I'm following a previous release. Do you have a checklist to share?

Reply:
Well, usually my checklist is the version + change log + tests.
I see we already have some parser tests - can I run them on github actions?